ansible/docker/Dockerfile.Alpine3

```diff
@@ -14,15 +14,11 @@ RUN set -eux; \
  rm -rf /ansible; apk del ansible
 
 ENV \
-    JDK7_BOOT_DIR="/usr/lib/jvm/zulu8" \ 
-    JDK8_BOOT_DIR="/usr/lib/jvm/zulu8" \
-    JDK10_BOOT_DIR="/usr/lib/jvm/zulu11" \
-    JDK11_BOOT_DIR="/usr/lib/jvm/zulu11" \
-    JDK14_BOOT_DIR="/usr/lib/jvm/zulu14" \
-    JDK15_BOOT_DIR="/usr/lib/jvm/zulu15" \
-    JDK16_BOOT_DIR="/usr/lib/jvm/zulu16" \
-    JDK17_BOOT_DIR="/usr/lib/jvm/zulu17" \
-    JDK18_BOOT_DIR="/usr/lib/jvm/zulu18" \
-    JDK19_BOOT_DIR="/usr/lib/jvm/zulu18" \
-    JDKLATEST_BOOT_DIR="/usr/lib/jvm/zulu18" \
-    JAVA_HOME="/usr/lib/jvm/zulu8"
+    JDK7_BOOT_DIR="/usr/lib/jvm/jdk-7" \ 
```

Comment:
Is this one going to be pointing anywhere useful? I'd be tempted to hold off a bit on these changes until we've verified they work properly. I get the impression the JDK11 in particular may be causing the hangs on the test jobs as some of the later versions don't seem to be freezing so much at the moment.

Reply:
yeah, I switched the Zulu 7 build to be installed there for now